apps/app_voicemail.c

```diff
@@ -11413,6 +11413,389 @@ static int vm_playmsgexec(struct ast_channel *chan, const char *data)
 	return 0;
 }
 
+static int show_mailbox_details(struct ast_cli_args *a)
+{
+#define VMBOX_STRING_HEADER_FORMAT "%-32.32s %-32.32s %-16.16s %-16.16s %-16.16s %-16.16s\n"
+#define VMBOX_STRING_DATA_FORMAT   "%-32.32s %-32.32s %-16.16s %-16.16s %-16.16s %-16.16s\n"
```

Comment:
The values for the 2 macros are identical, do they need to be separate?

Reply:
This is to be consistent with the other methods in case the line format changes and in case we add a concise/verbose option.